src/global/types/city.ts

```diff
@@ -1,4 +1,4 @@
-import { LocationType } from '..';
+import { LocationType } from '.';
```

Comment:
Не надо так писать, внутри 1 модуля ты можешь использовать относительные пути ./location'
```suggestion
import { LocationType } from './location';
```